resources/public/contracts/src/Offering.sol

```diff
@@ -81,6 +81,14 @@ contract Offering {
     }
 
     /**
```

Comment:
maybe we ought to add a meta-info map to the Offering contract for future updates?